cl/simple_pages/templates/help/relative_dates_help.html

```diff
@@ -0,0 +1,76 @@
+
+{% block description %}Learn how to use relative dates queries in CourtListener to make your searches and alerts more dynamic and up-to-date.{% endblock %}
+{% block og_description %}Learn how to use relative dates queries in CourtListener to make your searches and alerts more dynamic and up-to-date.{% endblock %}
+
+{% block sidebar %}{% endblock %}
+
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="col-xs-12 hidden-md hidden-lg">
+  <h4 class="v-offset-below-2">
+    <i class="fa fa-arrow-circle-o-left gray"></i>
+    <a href="{% url "help_home" %}">Back to Help</a>
+  </h4>
+</div>
+
+<div id="toc-container" class="hidden-xs hidden-sm col-md-3">
+  <div id="toc">
+    <h4 class="v-offset-below-3">
+      <i class="fa fa-arrow-circle-o-left gray"></i>
+      <a href="{% url "help_home" %}">Back to Help</a>
+    </h4>
+    <h3>Table of Contents</h3>
+    <ul>
+      <li><a href="#relative-dates">Relative Date Queries</a></li>
+      <li><a href="#syntax">Allowed Syntax</a></li>
+      <li><a href="#notes">Technical Notes</a></li>
+    </ul>
+  </div>
+</div>
+
+<div class="col-xs-12 col-md-8 col-lg-6">
+  <h1 id="relative-dates">Help with Relative Date Queries</h1>
+  <p class="lead">Use relative dates in your queries to keep your searches and alerts dynamically up to date.</p>
+  <p>When placing a search query into CourtListener, you can use either "Calendar" date queries or "Relative" date queries:</p>
+  <ul>
+    <li>Calendar date queries let you filter before or after a given date.</li>
+    <li>Relative dates filter results dynamically based on the date the query is made by a user or system. Instead of specifying a calendar date, such a "January 1, 2025", you can use relative time periods like <code>1 day ago</code>, <code>Past 1 month</code>, or <code>-14d</code>.</li>
+  </ul>
+
+  <p>Relative date queries are useful for saved or bookmarked queries, so that your query always filters to recent content.
+    They are also valuable when creating alerts. Imagine you want to receive an alert whenever a recent document — filed in the past week — is made available in the  <a href="{% url "advanced_r" %}">RECAP Archive</a>.
+    With relative dates, your date range is rolling, and the "Past Week" changes in sync with the date the alert is triggered.
+  </p>
+
+  <h2 id="syntax">Allowed Syntax</h2>
+  <p>You can use the following syntax:</p>
+  <ul>
+    <li><code>Xd ago</code></li>
+    <li><code>X days ago</code></li>
+    <li><code>-Xd</code></li>
+    <li><code>-Xd ago</code></li>
+    <li><code>-X days</code></li>
+    <li><code>Past X days</code></li>
+  </ul>
+
+  <p>To filter to months or years, substitute <code>m</code>, <code>months</code>, <code>y</code>, or <code>years</code> instead of <code>d</code> or <code>days</code>.</p>
+  <p>When using the Search API, relative dates are accepted on all date filters, even if they are not supported by the website.</p>
```

Comment:
Nice! This is much clearer to me, thanks! What do you think about linking to the Search API help page too? Makes it even clearer that it's that specific API and not just any of them:

```suggestion
  <p>When using the <a href="{% url "search_api_help" %}">Search API</a>, relative dates are accepted on all date filters, even if they are not supported by the website.</p>
```

And looking at that page I wonder, how do you feel about also adding a link _from_ the Search API help page back to this one? something like this:

```diff
- <p>To get the most out of this API, see our <a href="{% url "coverage" %}">coverage</a> and <a href="{% url "advanced_search" %}">advanced operators</a> documentation.</p>
+ <p>To get the most out of this API, see our <a href="{% url "coverage" %}">coverage</a>, <a href="{% url "advanced_search" %}">advanced operators</a>, and <a href="{% url "relative_dates" %}">relative date queries</a> documentation.</p>
```